packages/protocol/contracts/governance/Governance.sol

```diff
@@ -877,8 +877,12 @@ contract Governance is
     uint256 tally = 0;
     uint256 n = numberValidatorsInCurrentSet();
     for (uint256 idx = 0; idx < n; idx++) {
-      address validator = validatorAddressFromCurrentSet(idx);
-      if (isHotfixWhitelistedBy(hash, validator)) {
+      address validationSigner = validatorAddressFromCurrentSet(idx);
```

Comment:
Please pull in `asaj/hotfix-signer`, which renames this to `validatorSignerAddressFromCurrentSet`